theme/brb/views/imprint/imprint.hbs

```diff
@@ -0,0 +1,119 @@
+<div class="section-imprint">
```

Comment:
looks for duplicated file maybe use that file but import a single imprint.hbs instead

Reply:
Check, optimization later